### PR TITLE
Completed Javascript Arrays - Amelia Talkington

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-01-25 at 6 28 26 PM

### CODE REVIEW

File: javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var colors = ['red', 'white', 'blue'];
+console.log('Value of colors[0]:', colors[0]);
+console.log('Value of colors[1]:', colors[1]);
+console.log('Value of colors[2]:', colors[2]);
+
+var sentence = 'America is ' + colors[0] + ', ' + colors[1] + [', and '] + colors[2] + ['.'];
```
Comment (codingcodymiller) [inline javascript-arrays/main.js:6]: The string portions should not be in brackets

File: javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+console.log(sentence);
+
+colors[2] = 'green';
+var sentenceMexico = 'Mexico is ' + colors[0] + ', ' + colors[1] + [', and '] + colors[2] + ['.'];
```
Comment (codingcodymiller) [inline javascript-arrays/main.js:10]: The string portions should not be in brackets

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Amelia! Nice work so far, please see my notes

**Comment by ameliatalkington** [comment]
same output -- i can honestly say I have no idea what was going through my head wow

**Review by codingcodymiller — APPROVED** [review]
👍 👍 👍

**Pull request merged**